components/blocks/RecentPosts.tsx

```diff
@@ -23,30 +22,31 @@ export const RecentPostsBlock = ({ data, index, recentPosts }) => {
           </div>
         )}
         <div className="w-full flex flex-wrap gap-12">
-          { recentPosts.edges.map(({node: post}) => {
-
-            return (<DynamicLink href={`/`} passHref>
-            <a className="group flex-1 min-w-[20rem]">
-              <h3 className="font-tuner inline-block text-3xl lg:text-4xl lg:leading-tight bg-gradient-to-br from-blue-700/70 via-blue-900/90 to-blue-1000 group-hover:from-orange-300 group-hover:via-orange-500 group-hover:to-orange-700 bg-clip-text text-transparent mb-4">
-                {post.title}
-              </h3>
-              <RichTextWrapper>
-                <BlogMeta>
-                  <MetaBit>
-                    <strong>{formatDate(post.date)}</strong>
-                  </MetaBit>
-                  <MetaBit>
-                    <span>By</span> <strong>{post.author}</strong>
-                  </MetaBit>
-                </BlogMeta>
-                {getExcerpt(post.body, 200)}
-              </RichTextWrapper>
-            </a>
-          </DynamicLink>)
-          })
-          }
+          {recentPosts.edges.map(({ node: post }) => {
+            const slug = post['_sys']['filename']
```

Comment:
```suggestion
const slug = post._sys.filename
```
^ Can you do this? It will give you better typing

Reply:
Sure, looks a little cleaner. I'm not getting any type hints either way, though.